### PR TITLE
mpi incremental, first draft end-to-end

### PR DESCRIPTION
**Pull request by evan-phelps**
Includes pkg_cdw_incr.PROCESS_MPI_INCR and supporting tables.

Can be tested within any schema, but be careful, since the script DROPS/CREATES objects.

Intended to be deployed in the existing HSSC_ETL user with the following configuration: 

```
SELECT/INSERT/UPDATE/DELETE access to CDW tables
SELECT access to CDWREF
Private MPI db link
PKG_LOGGING
```

### CODE REVIEW

**Comment by evan-phelps** [comment]
Added basic HSSC_ETL user creation script.

**Comment by higgsmass** [comment]
I will test parts of this over the weekend and we can merge it Tuesday

File: scripts/sql/user_hssc_etl_create.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * After creating HSSC_ETL user, log in and create the private MPI DB Link:
+ 
+ CREATE DATABASE LINK "MPI"
```
Comment (higgsmass) [inline scripts/sql/user_hssc_etl_create.sql:4]: create database link MPI
  connect to my_schema identified by my_password
  using '(DESCRIPTION = 
    (ADDRESS = (PROTOCOL = TCP) (HOST = hssc-cdw-mpidb-d.clemson.edu)
    (PORT = 1521))
    (CONNECT_DATA = (SID = dtprd2))
    )';

File: scripts/sql/user_hssc_etl_create.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ CONNECT TO "PATIENT" IDENTIFIED BY <password>
+ USING 'hssc-cdw-mpidb-d:1521/mpidev'
+ ;
+ 
+ TODO: Orchestrate creation of user with grants and creation of private DB
+       link, which requires directly connecting as the new user.
+ */
+create user HSSC_ETL identified by changeme;
```
Comment (higgsmass) [inline scripts/sql/user_hssc_etl_create.sql:18]: can this be factored out into a procedure with HSSC_ETL as an argument to the procedure?

File: scripts/sql/user_hssc_etl_create.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+grant CREATE SESSION to HSSC_ETL;
+grant SELECT ANY TABLE to HSSC_ETL;
+grant CREATE DATABASE LINK to HSSC_ETL;
+
+BEGIN
+  FOR x IN (SELECT * FROM dba_tables WHERE owner in ('CDW'))
+  LOOP
+    EXECUTE IMMEDIATE 'GRANT SELECT, INSERT, UPDATE, DELETE ON '
```
Comment (higgsmass) [inline scripts/sql/user_hssc_etl_create.sql:33]: same here - pass privileges as arguments (maybe)

**Pull request merged**